meal-prep-frontend/src/App.js

```diff
@@ -1,10 +1,13 @@
 import './App.css';
 import { Header } from './components/Header';
+import { Day } from './components/Day';
+import { Week } from './components/Week';
 
 function App() {
   return (
     <div>
       <Header />
+      <Week />
```

Comment:
An improvement here would be some kind of body/middle section component which contains the week